client/components/learn/Resources/resources-model.ts

```diff
@@ -41,6 +41,11 @@ export const RESOURCES: Record<ResourceCategory, ResourceDescription[]> = {
       url: 'https://validator.w3.org/',
       cover: `${process.env.CLOUDFRONT_PUBLIC}/public/images/resources/w3c-validator_500w.png`,
     },
+    {
+      title: 'Patterns Dev',
```

Comment:
Please move in the books section. Put it in second place, after "You Don't Know JS"